lrrbot/commands/static.py

```diff
@@ -1,86 +1,143 @@
+import itertools
 import logging
 import aiomas
 import random
 import re
 import irc.client
+import sqlalchemy
 
 import lrrbot.decorators
 from common.config import config
-from lrrbot import storage
+import common.utils
 from lrrbot.command_parser import Blueprint
 
 blueprint = Blueprint()
 log = logging.getLogger(__name__)
 
-def generate_docstring():
-	inverse_responses = {}
-	for command, data in storage.data["responses"].items():
-		response = data["response"]
-		if isinstance(response, (tuple, list)):
-			response = tuple(response)
-		inverse_responses.setdefault((response, data["access"]), [])
-		inverse_responses[(response, data["access"])] += [command]
+ADVICE = 1
+
+ACCESS_ANY = 0
+ACCESS_SUB = 1
+ACCESS_MOD = 2
+
+def _get_command_id(bot, conn, command):
+	if isinstance(command, int):
+		return command
+	else:
+		aliases = bot.metadata.tables["commands_aliases"]
+		query = sqlalchemy.select(aliases.c.command_id).where(aliases.c.alias == command)
+		row = conn.execute(query).first()
+		if row is None:
+			return None
+		else:
+			return row[0]
```

Comment:
This is [`Result.scalar()`](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Result.scalar).
```suggestion
		return conn.execute(query).scalar()
```